chat/src/main/java/net/md_5/bungee/chat/ComponentSerializer.java

```diff
@@ -158,11 +159,27 @@ public static JsonElement toJson(ComponentStyle style)
         return gson.toJsonTree( style );
     }
 
+    /**
+     * @param object the object to serialize
+     * @return the JSON string representation of the object
+     * @deprecated Error-prone, be careful which object you input here
+     */
+    @Deprecated
     public static String toString(Object object)
     {
         return gson.toJson( object );
     }
 
+    public static String toString(Content content)
+    {
+        return gson.toJson( content );
+    }
+
+    public static String toString(JsonElement element)
```

Comment:
Where is this used?

Reply:
Actually it is not used, oops. But it refers to the correct overload of gson for json elements.